api/machines_test.go

```diff
@@ -136,3 +137,126 @@ func TestExtractMachinePage(t *testing.T) {
 		}
 	}
 }
+
+func TestMachinesPatchAddModify(t *testing.T) {
+	reqBody := `
+	[{"op": "add", "path": "/XXX/metadata/foo", "value": "bar"},
+	 {"op": "replace", "path": "/YYY/metadata/ping", "value": "splat"}]
+	`
+
+	resource, rw := fakeMachinesSetup()
```

Comment:
Build of unit test fails at this line as well as other 4 other call sites, because ``fakeMachinesSetup()`` is undefined. In the previous PR #1077 , the function actually did exist.